### PR TITLE
run e2e tests on fasterci

### PR DESCRIPTION
**Pull request by apesternikov**

### CODE REVIEW

File: .fasterci/config.yaml [hunk 1]
```diff
@@ -16,3 +16,31 @@ workflows:
                     - //...
                 test_targets:
                     - //...
+          - name: e2e/link_js_package
```
Comment (alexeagle) [inline .fasterci/config.yaml:19]: hey, thanks - could we use the same matrix syntax as in the github actions configuration?

Comment (alexeagle) [reply .fasterci/config.yaml:19]: this is what I'm referring to: https://github.com/aspect-build/rules_js/blob/main/.github/workflows/ci.yaml#L19-L25

**Comment by gregmagolan** [comment]
I hit a failure on unit tests on FasterCI that was consistent and only seen there and not our github actions CI. Turned off FasterCI so it wouldn't break master. See https://github.com/aspect-build/rules_js/commit/cd7dafad8b3c86ff7ff337c50259721949dfdea2

File: .fasterci/config.yaml (hunk 1 above)
Comment (apesternikov) [reply .fasterci/config.yaml:19]: Using yaml refs is recommended way to achieve this. see https://github.com/adobe/rules_gitops/blob/fd76291f513fce4fc637c22f8055b3a49ad82fe7/.fasterci/config.yaml#L32 for example.

**Pull request closed without merging**